build/charts/antrea/values.yaml

```diff
@@ -141,6 +141,9 @@ antreaProxy:
   # will only handle Services without the "service.kubernetes.io/service-proxy-name"
   # label, but ignore Services with the label no matter what is the value.
   serviceProxyName: ""
+  # -- Determines how external traffic's processed when it's load balanced across nodes. It must be one of "nat" or
+  # "dsr".
+  loadBalanceMode: "nat"
```

Comment:
for some reason, I thought we were going to have the ability to enable DSR on a per-Service basis.
is that not feasible, or do you think it is not interesting to have that granularity?